source/renderer/util/table-helper.js

```diff
@@ -119,12 +119,16 @@ class TableHelper {
     for (let i = 0; i < ast.length; i++) {
       let row = $('<tr></tr>').css('width', '100%')
       this._elem.append(row)
-      for (let j = 0; j < ast[i].length; j++) {
-        // Preset the colSizes
-        if (!this._colSizes[j]) this._colSizes[j] = ast[i][j].length
-        if (ast[i][j].length > this._colSizes[j]) this._colSizes[j] = ast[i][j].length
-        row.append($('<td></td>').text(ast[i][j]).css('text-align', colAlignments[j]).attr('contenteditable', 'true'))
-      }
+      ast[i].forEach((cell, index) => {
+        this._colSizes[index] = Math.max( // Finding the widest cell
+          cell.length,
+          this._colSizes[index] || 0 // default to 0
+        )
+
+        const template = document.createElement('template')
```

Comment:
I used the `<template>` HTML tag as it expresses clear the intent of this block: render a HTML string into DOM elements. Interestingly, HTML `<template>` tags could be [in the HTML file directly](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template).

As for the clone clone: to avoid having a reference to the parent element and avoid accidental mutation in the future. It is probably OK to skip the cloning here as the element newly created, however, I was concerned that the garbage collector won't be able to free up the parent element.

Reply:
It is used in 3 places now, I'll move it until `util/`.

Reply:
Done.